src/lib/operators/get_table.hpp

```diff
@@ -18,12 +20,15 @@ class GetTable : public AbstractReadOnlyOperator {
 
   const std::string& table_name() const;
 
+  void set_excluded_chunks(const std::vector<ChunkID>& excluded_chunks);
```

Comment:
All other operators take all of their config as constructor param, GetTable shouldn't be different.